docs/usage.md

```diff
@@ -1,62 +1,73 @@
+        - Sigma Rules: Sigma Rules provide a standard format for log events. They are helpful in searching or pattern matching through log data.
+        
+            Using Sigma Rules we can create signature-based Threat Hunt Rules for various Threat Detection Integrations like SIEM, Analyzer, or EDR.
+        
+        - Fortinet Fabric Rules: With every Outbreak Alert FortiGuard provides FortiAnalyzer and FortiSIEM as part of Fortinet fabric solution.
+
+4. Create CVEs for KEVs: Using NVD integration FortiSOAR checks if an associated CVE is tagged as a KEVs. Once found, it creates the CVE records in the vulnerability module and links those records to outbreak alerts.
+
+5. Ingest IOCs as Threat Feeds: IOCs associated with the Outbreak are ingested as threat feeds in FortiSOAR.
 
-5. Ingest IOCs as Threat Feeds: IOCs associated to the Outbreak are ingested as threat feeds in FortiSOAR Say, a new IOC is added to the same outbreak, user does not need to update the solution pack, as we will be hosting the IOCs on our public github page, where our team will be updating them. so threat hunting queries always get the latest IOCs.
+    IOCs are hosted on our public GitHub page and regularly updated by the FortiSOAR team so the threat hunting queries have the latest IOCs. These regular updates avoid the need to update the solution pack if a new IOC is added to the same outbreak.
 
-If any other ALERT in FortiSOAR containing those IOCs is found, user is notified and severity is raised.
+    Users are notified and the alert severity is raised if an alert in FortiSOAR containing these IOCs is found.
 
-6. IOC Threat Hunt:
+6. You can perform IOC Threat Hunt, and create outbreak IOC hunt alerts, using any of the following:
 
-    a. Perform IOC Threat Hunting using Fortinet Fabric solutions (FortiSIEM/FortiAnalyzer) and creates Outbreak IOC Hunt Alerts in FortiSOAR
+    a. Fortinet Fabric solutions (FortiSIEM/FortiAnalyzer)
 
-    b. Perform IOC Threat Hunting using SIEM solutions (QRadar/Splunk) and creates Outbreak IOC Hunt Alerts in FortiSOAR
+    b. SIEM solutions (QRadar/Splunk)
 
-7. Signature Based Threat Hunt (Sigma Rules):
+7. Signature Based Threat Hunt using Sigma Rules:
 
-    a.Perform Signature Based Threat Hunting using Fortinet Fabric solutions (FortiSIEM/FortiAnalyzer) and creates Outbreak Alerts in FortiSOAR
+    a.Perform Signature-based Threat Hunting using Fortinet Fabric solutions (FortiSIEM/FortiAnalyzer) and create Outbreak Alerts in FortiSOAR
     ![FortiSIEM Fabric Rule](./res/fsm_fortinet_fabric.png)
 
     ![FortiAnalyzer Fabric Rule](./res/faz_fortinet_fabric.png)
 
-    b. Perform Signature Based Threat Hunting using SIEM solutions (QRadar/Splunk/Azure Log Analytics) and creates Outbreak Alerts in FortiSOAR.
+    b. Perform Signature-based Threat Hunting using SIEM solutions (QRadar/Splunk/Azure Log Analytics) and create Outbreak Alerts in FortiSOAR.
```

Comment:
Foritisiem and fortianalyzer can also be added.

Reply:
Fixed